#! /usr/bin/env python
# Task Long configuration

PACKAGE='cb_tracker'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("kp_x",    double_t, 0,    "Slew gain",  1.0, -10.0, 10.0)
gen.add("zoom_setpt", double_t, 0,    "Zoom size",  0.20, 0.01, 0.99)
gen.add("kp_zoom", double_t, 0,    "Zoom gain", 1.0, -20.0, 20.0)

exit(gen.generate(PACKAGE, "cb_tracker_cfg", "CameraControl"))

